<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepCount</class>
 <widget class="QMainWindow" name="DeepCount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>830</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>830</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeepCount</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="classify_tab">
       <attribute name="title">
        <string>Classify</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCustomPlot" name="trace_plot" native="true">
            <property name="minimumSize">
             <size>
              <width>684</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCustomPlot" name="x_plot" native="true"/>
            </item>
            <item>
             <widget class="QCustomPlot" name="y_plot" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="step_count_group">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>330</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Steps</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="zero_step">
               <property name="text">
                <string>0 Step</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="one_step">
               <property name="text">
                <string>1 Step</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="two_step">
               <property name="text">
                <string>2 Step</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="three_step">
               <property name="text">
                <string>3 Step</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="four_step">
               <property name="text">
                <string>4 Step</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="five_step">
               <property name="text">
                <string>5+ Step</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="skip_step">
               <property name="text">
                <string>Skip</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>one_step</zorder>
            <zorder>two_step</zorder>
            <zorder>three_step</zorder>
            <zorder>four_step</zorder>
            <zorder>five_step</zorder>
            <zorder>skip_step</zorder>
            <zorder>zero_step</zorder>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="previous">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Previous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="next">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="evaluate_tab">
       <attribute name="title">
        <string>Evaluate</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
